### PR TITLE
`test_cq_compile` in integration test hotfix

### PR DESCRIPTION
**Pull request by pgoiporia**

### CODE REVIEW

**Review by perlinm — APPROVED** [review]
LGTM after fixing format

**Review by vtomole — COMMENTED** [review]
Do we just want to check with `isinstance` similar to https://github.com/SupertechLabs/cirq-superstaq/blob/7d0af0c368f99efa7edac2d2591513d8b117f2d9/cirq_superstaq/daily_integration_test.py#L146? This circuit might change again later and the daily integration test will be failing for something that it's not really supposed to fail for.

**Comment by pgoiporia** [comment]
> Do we just want to check with `isinstance` similar to https://github.com/SupertechLabs/cirq-superstaq/blob/7d0af0c368f99efa7edac2d2591513d8b117f2d9/cirq_superstaq/daily_integration_test.py#L146? This circuit might change again later and the daily integration test will be failing for something that it's not really supposed to fail for.

thoughts @perlinm ?

**Comment by perlinm** [comment]
I agree, we should.  May as well do the same in for the rest of the compile methods in this test

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -116,9 +116,8 @@ def test_cq_compile(provider: qss.SuperstaQProvider) -> None:
     circuit = qiskit.QuantumCircuit(1)
     circuit.h(0)
     expected = qiskit.QuantumCircuit(6)
```
Comment (perlinm) [inline qiskit_superstaq/daily_integration_test.py:118]: Something like this maybe?
```suggestion
    assert isinstance(provider.cq_compile(circuit).circuit, cirq.Circuit)
    circuits = provider.cq_compile([circuit]).circuits
    assert len(circuits) == 1 and isinstance(circuits[0], cirq.Circuit)
    circuits = provider.cq_compile([circuit, circuit]).circuits
    assert len(circuits) == 2 and isinstance(circuits[0], cirq.Circuit) and isinstance(circuits[1], cirq.Circuit)
```

Comment (vtomole) [reply qiskit_superstaq/daily_integration_test.py:118]: Yep.

File: qiskit_superstaq/daily_integration_test.py [hunk 2]
```diff
@@ -115,7 +115,11 @@ def test_cq_compile(provider: qss.SuperstaQProvider) -> None:
 
     circuit = qiskit.QuantumCircuit(1)
     circuit.h(0)
-    expected = qiskit.QuantumCircuit(6)
+    assert isinstance(provider.cq_compile(circuit).circuit, cirq.Circuit)
+    circuits = provider.cq_compile([circuit]).circuits
+    assert len(circuits) == 1 and isinstance(circuits[0], cirq.Circuit)
+    circuits = provider.cq_compile([circuit, circuit]).circuits
+    assert len(circuits) == 2 and isinstance(circuits[0], cirq.Circuit) and isinstance(circuits[1], cirq.Circuit)
     expected.append(GR(1, 0.5 * np.pi, -0.5 * np.pi), [0])
```
Comment (vtomole) [inline qiskit_superstaq/daily_integration_test.py:123]: Get rid of anything that uses this `expected` variable.

Comment (pgoiporia) [reply qiskit_superstaq/daily_integration_test.py:123]: @perlinm I don't think qss is supposed to use cirq

Comment (perlinm) [reply qiskit_superstaq/daily_integration_test.py:123]: Oh then maybe that should be a `qiskit.QuantumCircuit` or something

Comment (vtomole) [reply qiskit_superstaq/daily_integration_test.py:123]: Yeah that's right. The example @perlinm posted wasn't supposed to be the the exact thing we should commit. It should be similar in spirit; that is use `qiskit.Circuit` to compare it.

**Review by vtomole — COMMENTED** [review]
Did you run this against the local server and it passed?

**Comment by pgoiporia** [comment]
> 

yep!

**Review by vtomole — APPROVED** [review]
Nice, thanks!

**Pull request merged**